packages/kolibri-common/components/SyncSchedule/ManageSyncSchedule.vue

```diff
@@ -261,11 +276,49 @@
       },
       editButton(value) {
         if (value !== ' ') {
-          this.$router.push({ name: PageNames.EDIT_SYNC_SCHEDULE, params: { deviceId: value } });
+          this.$router.push({
```

Comment:
The fact that some of our routes are using `facility_id` as the facility parameter, means that this isn't going to work universally.